backend/kernelCI_app/constants/localization.py

```diff
@@ -13,3 +13,35 @@ class ClientStrings:
     TREE_BOOTS_NOT_FOUND = "No boots available for this tree/branch/commit"
     TREE_NOT_FOUND = "No results available for this tree/branch/commit"
     TREE_TESTS_NOT_FOUND = "No tests available for this tree/branch/commit"
+    TREE_COMMITS_HISTORY_NOT_FOUND = "History of tree commits not found"
+    TREE_STATUS_HISTORY_NOT_FOUND = "Test status history not found"
+    TEST_NO_ISSUE_FOUND = "No issues were found for this test"
+    TEST_NOT_FOUND = "Test not found"
+    PROXY_FETCH_FAILED  = "Failed to fetch resource:"
+    PROXY_INVALID_URL = "Invalid URL"
+    PROXY_ERROR_FETCH = "Error fetching the resource"
+    ORIGIN_NOT_FOUND = "No origins found"
+    LOG_TABLE_NOT_FOUND = "Table with id 'list' not found"
+    LOG_TABLE_BODY_NOT_FOUND = "Table body not found"
+    LOG_INVALID_TABLE_FORMAT = "Invalid number of columns in table row (probably not a log website)"
+    LOG_NO_FILES_FOUND = "No log files found"    
+    ISSUE_NOT_FOUND = "No issues found"
+    ISSUE_INVALID_JSON = "Invalid body, request body must be a valid json string"
```

Comment:
This is not really related to the issue details, and it is also shared by the hardware endpoint, so you can remove the `ISSUE_` prefix and use it on all endpoints that use this string
```suggestion
    INVALID_JSON_BODY = "Invalid body, request body must be a valid json string"
```